src/get-element/getElement.js

```diff
@@ -50,11 +50,10 @@ const getElement = (cssSelector, outTimer = 10000, onError = null) => {
       const errorMessage = `Timeout while waiting for ${cssSelector}`;
       if (onError && typeof onError === 'function') {
         onError(errorMessage);
+        reject();
```

Comment:
Why not pass the error message here to? Seems like we could not include the reject here, leave the reject on line 57, and get rid of the else block entirely.

Reply:
I've resolved the issue you pointed out. The error message is now always passed to the reject function, and I've removed the else block to simplify the logic. Thanks for the feedback!